x-pack/solutions/security/plugins/security_solution_ess/public/plugin.ts

```diff
@@ -30,6 +31,10 @@ export class SecuritySolutionEssPlugin
     _core: CoreSetup,
     _setupDeps: SecuritySolutionEssPluginSetupDeps
   ): SecuritySolutionEssPluginSetup {
+    const { securitySolution } = _setupDeps;
```

Comment:
The leading underscore in the `_setupDeps` means the variable is not used, can we rename it to `setupDeps`?